docs/streams/upgrade-guide.html

```diff
@@ -232,6 +232,11 @@ <h3><a id="streams_api_changes_280" href="#streams_api_changes_280">Streams API
         Kafka Streams throws a <code>TimeoutException</code>
         (cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-572%3A+Improve+timeouts+and+retries+in+Kafka+Streams">KIP-572</a>).
     </p>
+    <p>
+        We changed <code>default.key.serde</code> and <code>default.value.serde</code> to be <code>null</code> instead of <code>ByteArraySerde</code>.
```

Comment:
```suggestion
        We changed the default value of <code>default.key.serde</code> and <code>default.value.serde</code> to be <code>null</code> instead of <code>ByteArraySerde</code>.
```